docs/contribution-guide.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: "Contribution guide"
+---
+
+Thank you for interest in contributing to Ship. Here, you'll find all the necessary information to get started.
+
+## Ways to contribute
+
+- **Fix Ship docs** – fixing issues like bad wording, incomplete information, or missing examples in the documentation.
+- **Fix Ship template** - contribute to resolving bugs and enhancing the existing code in current template.
+- **Give feedback and suggest improvements** - report bugs through [issues](https://github.com/paralect/ship/issues) or suggest improvements.
+- **Share Ship** - share Ship link with everyone how can be interested.
+
+## How to open pull request
+1. Fork the [project](https://github.com/paralect/ship).
+2. Set up the project locally.
+3. Create a new branch from the `master` branch, using a name that describes your changes.
+4. Implement your changes and create a commit.
+5. Open a pull request.
+
+## Documentation
+
+We use [Mintlify](https://mintlify.com/) for documentation, located in the `docs/` folder.
+
+The docs are written in [MDX](https://mdxjs.com/) format. Check out the [Github Markdown Guide](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
+and [Mintlify documentation](https://mintlify.com/docs/quickstart) to become familiar with this syntax.
+
+### How to run documentation
+
+1. Install Mintlify globally.
+
+<Warning>
+  Ensure that your Node.js version is 19.00.0 or higher.
+</Warning>
+
+```shell
+pnpm i -g mintlify
+```
+
+2. Execute the following command in the project's root directory:
+
+```shell
+pnpm run docs
+```
+
+Documentation will be opened on [http://localhost:3000](http://localhost:3000) if it's not in use.
+
+## Ship template structure
```

Comment:
write about `/packages` folder
just overview, that we have cli tool code in `create-ship-app` folder
and take description from readme and the same with `node-mongo`